test/source/tests/setup.ts

```diff
@@ -992,6 +992,19 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
         await dbPage.close();
       }));
 
+    ava.default('user@custom-sks.flowcrypt.test - Respect custom key server url', testWithBrowser(undefined, async (t, browser) => {
+      const acct = 'user@custom-sks.flowcrypt.test';
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acct);
+      await SetupPageRecipe.autoSetupWithEKM(settingsPage);
+      const composePage = await ComposePageRecipe.openStandalone(t, browser, acct);
+      await ComposePageRecipe.fillMsg(composePage, { to: 'test@custom-sks.flowcrypt.test' }, 'Respect custom key server url');
+      await composePage.waitForContent('.email_address.has_pgp', 'test@custom-sks.flowcrypt.test');
```

Comment:
If you try to actually send a message with this key, it turns out to be invalid for encryption. I created #4833. So this key isn't really `has_pgp`, how did you created it? Save this key to test the implementation of #4833

Reply:
Nothing special. Created key with GPG Keychain (with no expiration)

Reply:
Nice catch though

Reply:
Hmmm.. @rrrooommmaaa I tried to reproduce above issue you mentioned but I couldn't.
As I debugged, it says usableForEncryption is true.
How did you reproduce it?

![image](https://user-images.githubusercontent.com/77344191/206702328-65eaed23-a384-4634-91be-20d77311bad9.png)

Reply:
when pressing "Send" button, the key is loaded from the storage, validation happens there

Reply:
I noticed too that sometimes validation is obviously skipped (the key is usable), so that's another point of concern

Reply:
When pressing send button, i get another error.
![image](https://user-images.githubusercontent.com/77344191/206709738-0ce64a35-c65b-4be6-b63e-dd6eb4a9a85d.png)

Reply:
Right, I'm getting the same now but I got different thing in the morning!
Looks like this is issue #4782 again!
The key creation time is 1670482106000 which is 	Thu Dec 08 2022 06:48:26 GMT (yesterday)
I suppose this is the time when you created the key? Why wasn't the key valid this morning for me? Hmmm...
Maybe it's a bug specific to OpenPGP v4
Looks like the key is invalid for about 24 hours after creation??

Reply:
Yes
> I suppose this is the time when you created the key?
Not sure
> Looks like the key is invalid for about 24 hours after creation??

Reply:
But I don't think it's invalid for 24 hours after creation.
Test worked for me & semaphoreci side

Reply:
I tried to create new key and test with that key but it works fine with new key.

Reply:
Could you try to replicate above issue you mentioned?

Reply:
I guess we can merge this PR, and I'm not sure what the system clock was on my virtual machine in the morning -- it may have remained back in the past before I rebooted it, since testing of #4782
Anyway, issue #4833 is still relevant -- the key wasn't usable and was green.
And we have no means to inform the user that an imported pubkey has a date in the future, which isn't convenient

Reply:
Looks like my system clock was indeed in the past